### ISSUE TITLE
The tiny model gives poor results compared to darknet detector (using the same weights)

### ISSUE THREAD

**Issue by amineHorseman** [qqwweee/keras-yolo3#234]
I'm trying to investigate the difference between this code and the original yolo model from darknet.

I trained the tiny model using darknet and I got good accuracy results using the darknet detector.
When I used the same weights using the keras_yolo3 code, I got lot of non detected objects.

I know that there's a small difference between the layers, but I don't think this have a huge impact on the accuracy. I suspect something else is influencing the results.

**Comment by karan1508** [comment]
@amineHorseman I am facing a similar issue. Did you find any solution?

**Comment by amineHorseman** [comment]
I didn't @karan1508, 
I just retrained the model from scratch

**Comment by UndecidedBoy** [comment]
Hello,
After retraining the net, did you get similar results to the original implementation in darknet? (Similar mAP, loss function converging at a similar rate as the original in terms of epochs or did it take more epochs to converge?)
Have a nice day!

**Comment by zzks** [comment]
I trained a yolov3 tiny model by darknet, then convert it, inference by this, works very poor !
